VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -470,4 +479,12 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseDirective implemen
             this.leaveJudicialConsultation();
         }
     }
+    async joinInSessionHearing() {
```

Comment:
would you mind rename this to callParticipantIntoHearing or something because joinHearingInSession and joinInSessionHearing are quite similar